.markdownlint-cli2.mjs

```diff
@@ -0,0 +1,15 @@
+import { init } from "./index.js";
+
+const configOptions = await init({
```

Comment:
I was about to recommend pointing of our projects to this branch to make sure things run as expected with this big change/new setup, but it seems like this basically acts like a test? Would it be worth testing this out in one of our larger repos?

Reply:
Yeah, my understanding after reading for awhile is that we only have this file/use `markdownlint-cli2` so we can [run our own linting setup](https://github.com/github/markdownlint-github/blob/main/package.json#L12) against the files in this repo (meta-testing, I guess).